creusot-contracts-dummy/src/lib.rs

```diff
@@ -106,3 +112,36 @@ pub fn derive_deep_model(_: TS1) -> TS1 {
 pub fn derive_resolve(_: TS1) -> TS1 {
     TS1::new()
 }
+
+struct DeleteInvariants;
+
+impl VisitMut for DeleteInvariants {
+    fn visit_expr_for_loop_mut(&mut self, i: &mut syn::ExprForLoop) {
+        delete_invariants_attrs(&mut i.attrs);
+        syn::visit_mut::visit_expr_for_loop_mut(self, i)
+    }
+
+    fn visit_expr_while_mut(&mut self, i: &mut syn::ExprWhile) {
+        delete_invariants_attrs(&mut i.attrs);
+        syn::visit_mut::visit_expr_while_mut(self, i)
+    }
+
+    fn visit_expr_loop_mut(&mut self, i: &mut syn::ExprLoop) {
+        delete_invariants_attrs(&mut i.attrs);
+        syn::visit_mut::visit_expr_loop_mut(self, i)
+    }
+}
+
+fn delete_invariants_attrs(attrs: &mut Vec<syn::Attribute>) {
+    attrs.retain(|attr| {
+        if let syn::Meta::List(meta) = &attr.meta {
+            !meta.path.is_ident("invariant")
```

Comment:
should we handle `#[creusot_contracts::invariant(...)]` too?

Reply:
I guess? Is it significantly more difficult?

Reply:
no, I just wanted to be sure.